data/seed/Seeding.md

```diff
@@ -0,0 +1,216 @@
+  of public body data:
+
+    ```
+    $ python manage.py loaddata data/seed/test-categories.json
+    ```
+- Run this command to load the subset of public body data:
+
+    ```
+    $ python manage.py import_csv data/seed/test-public-bodies.csv
+    ```
+
+#### Loading public bodies via the Admin website
+
+You can also use the Admin website to load public bodies from a CSV file.
+
+- On the `Public Body` page (Home > Public Body > Public Bodies), scroll
+  down to the bottom of the page to where there is a `Choose File` button next
+  to the `Import Public Bodies` button.
+
+- Click on the `Choose File` button and navigate to the directory with
+  your CSV file. Select that file and click on the `Open` button.
+
+- Back on the `Public Body` page, click on the `Import Public Bodies` button.
+
+- If this works, you should see a message on the `Public Body` page that says
+  that the public bodies were imported.
+
+
+### Loading all public bodies
+
+If the first test loading of the database with the very small set of data works,
+you can load the full set of public bodies.
+
+- Stop your Django server by pressing Ctl-C in the terminal window running the
+  server.
+
+- Run these commands to re-initialize the database:
+
+    ```
+    $ sh data/seed/clear_db.sh
+    $ sh data/seed/init_db.sh
+    ```
+
+- To load a full set of data, you can use these files:
+    - 2024-03-24-categories.json
+    - 2024-03-24-public-bodies-fixed.csv
+
+- Load the categories:
+    ```
+    $ python manage.py loaddata data/seed/2024-03-24-categories.json
+    ```
+- Load the public bodies:
+    ```
+    $ python manage.py import_csv data/seed/2024-03-24-public-bodies-fixed.csv
+    ```
+
+At this point, you should have all of the basic data from UIPA.org without any
+requests and other data.
+
+*NOTE:*
+
+> When loading the public bodies from a CSV file, not all of them are loaded.
+> There's a bug in the CSV importer that messes up the slug for the name. This
+> causes public bodies with parents to not be loaded and stops with an error
+> message about "PublicBody matching query does not exist".
+>
+> To overcome this, on the Admin website, delete all of the public bodies
+> loaded and load the public bodies from the CSV file. Repeat this process a
+> couple more times until you see all 201 public bodies loaded. Spot check that
+> the slug matches the name by visiting the details of a public body.
+
+
+### Convenience scripts
+
+For convenience, you run these shell scripts to load the fixture files and
+public bodies:
+
+```
+$ sh data/seed/load_fixtures.sh
+$ sh data/seed/load_public_bodies.sh
+```
+
+## Preparing a CSV file to upload public bodies
+
+If you're going to be using a different CSV extract of public bodies, you'll
+need to:
+
+- Extract the tag values from the old `tags` field in the exported public
+  bodies CSV file.
+
+    ```
+    $ python extract_sets.py ../2024-03-15-Hawaii_UIPA_Public_Bodies_All.csv
+    ```
+
```

Comment:
nit: The formatting here is off:

<img width="1063" alt="Screenshot 2024-03-27 at 7 33 18 PM" src="https://github.com/CodeWithAloha/uipa/assets/15609358/5e66cf35-70ad-4bf7-b3af-4c106d58da23">